src/domain/event/eventHero/EventHero.tsx

```diff
@@ -111,10 +115,11 @@ const EventHero: React.FC<Props> = ({ event, eventType }) => {
                 <div className={styles.description}>{shortDescription}</div>
               )}
               <Visible above="sm" className={styles.date}>
-                {!!startTime &&
+                {superEvent !== undefined &&
```

Comment:
I didnt test but quickly it looks like that not any date range is shown when superevent is not defined? Shouldnt it default the to the event time then?

Reply:
Also it seems that no test goes red because of this. There should be a two tests on EventHero with and without the superEvent and both checks that the value of dateRange is correct

Reply:
Yup it seems to be a regression bug introduced, so tests should be added.
![Screenshot 2021-02-16 at 2 18 46 PM](https://user-images.githubusercontent.com/7648194/108062372-65165380-7062-11eb-9fb2-fca3289aed40.png)

Reply:
i will check

Reply:
fixed